packages/babel/package.json

```
@@ -13,10 +13,15 @@
   "author": "Rich Harris",
   "homepage": "https://github.com/rollup/plugins/tree/master/packages/babel#readme",
   "bugs": "https://github.com/rollup/plugins/issues",
-  "main": "dist/index.js",
-  "module": "dist/index.es.js",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/es/index.js",
+  "exports": {
+    "import": "./dist/es/index.js",
+    "types": "./types/index.d.ts",
```

It won't matter here but the TS team recommendation is to put `types` first:
```suggestion
    "types": "./types/index.d.ts",
    "import": "./dist/es/index.js",
```